File: tutorias-inf-uva-api/src/main/java/es/uva/inf/tutorias/business/domain/models/TipoHorario.java
package es.uva.inf.tutorias.business.domain.models;

import java.io.Serializable;
import java.time.LocalDate;


/**
 * The persistent class for the tipohorario database table.
 * 
 */
public class TipoHorario implements Serializable {
	private static final long serialVersionUID = 1L;

	private short id;
	private String tipo;
	private LocalDate fechaInicio;
	private LocalDate fechaFin;

	public TipoHorario() {
	}

	public short getId() {
		return this.id;
	}

	public void setId(short id) {
		this.id = id;
	}

	public String getTipo() {
		return this.tipo;
	}

	public void setTipo(String tipo) {
		this.tipo = tipo;
	}

	public LocalDate getFechaInicio() {
		return fechaInicio;
	}

	public void setFechaInicio(LocalDate fechaInicio) {
		this.fechaInicio = fechaInicio;
	}

	public LocalDate getFechaFin() {
		return fechaFin;
	}

	public void setFechaFin(LocalDate fechaFin) {
		this.fechaFin = fechaFin;
	}

}

File: tutorias-inf-uva-frontend/src/app/models/tipo-horario.ts
export class TipoHorario {
	id?: number;
	tipo?: string;
	fechaInicio?: Date;
	fechaFin?: Date;

	constructor(id?: number, tipo?: string, fechaInicio?: Date, fechaFin?: Date) {
		if (id) {
			this.id = id;
		}

		if (tipo) {
			this.tipo = tipo;
		}

		if (fechaInicio) {
			this.fechaInicio = fechaInicio;
		}

		if (fechaFin) {
			this.fechaFin = fechaFin;
		}
	}
}